color.go

```diff
@@ -91,6 +96,8 @@ func (p Profile) Color(s string) Color {
 	var c Color
 	if strings.HasPrefix(s, "#") {
 		c = RGBColor(s)
+	} else if _, ok := palette.CSS.Color(s); ok {
```

Comment:
I think this should be the last resort and we should try and convert the string to an integer first and, if that succeeds, treat it as a regular ANSI color. Not that there's currently a clash, but if someone added "8" as a color name to that palette, it would override the ANSI color. We wouldn't want that.

Reply:
Makes sense. Updated.